File: test/blackbox/fixtures/sdk_go_real/sykli.go
//go:build ignore

package main

import sykli "github.com/yairfalse/sykli/sdk/go"

func main() {
	s := sykli.New()

	s.Task("lint").Run("echo lint ok")
	s.Task("test").Run("echo test ok")
	s.Task("build").Run("echo build ok").After("lint", "test")

	s.Emit()
}

File: test/blackbox/fixtures/sdk_rust_real/sykli.rs
use sykli::Pipeline;

fn main() {
    let mut p = Pipeline::new();

    p.task("lint").run("echo lint ok");
    p.task("test").run("echo test ok");
    p.task("build").run("echo build ok").after(&["lint", "test"]);

    p.emit();
}
